src/utils/import-apply-types-and-projection.js

```diff
@@ -116,8 +116,14 @@ export function transformProjectedTypesStream({ transform = [], exclude = [], re
   return new Transform({
     objectMode: true,
     transform: function(doc, _encoding, cb) {
-      const result = transformProjectedTypes(doc, { transform, exclude, removeBlanks });
-      cb(null, result);
+      try {
+        const result = transformProjectedTypes(doc, { transform, exclude, removeBlanks });
+        cb(null, result);
+        return;
```

Comment:
you shouldn't need the `return`, just callback should be fine. Did the transform not work without the return at all?

Reply:
Ah eslint was complaining. Moved it to returning the callback though to be a little cleaner.